.rubocop.yml

```diff
@@ -109,6 +109,8 @@ Metrics/BlockNesting:
 Metrics/ClassLength:
   CountComments: false
   Max: 132 # TODO: Lower to 100
+  Exclude:
+    - "lib/rails_admin/config.rb"
```

Comment:
Thank you for taking care, but this issue got a different fix so please restore.
https://github.com/railsadminteam/rails_admin/commit/dec5be4fc48c1e626ef91796be0cb9fb61da1f7b